pkg/model/participation/participation_store.go

```diff
@@ -520,3 +541,81 @@ func (pm *ParticipationManager) increaseStakingRewardForEventAndAddress(eventID
 
 	return mutations.Set(stakingKeyForEventAndAddress(eventID, addressBytes), m.Bytes())
 }
+
+func totalParticipationStakingKeyForEventPrefix(eventID EventID) []byte {
+	m := marshalutil.New(37)
+	m.WriteByte(ParticipationStoreKeyPrefixStakingTotalParticipation) // 1 byte
+	m.WriteBytes(eventID[:])                                          // 32 bytes
+	return m.Bytes()
+}
+
+func totalParticipationStakingKeyForEvent(eventID EventID, milestone milestone.Index) []byte {
+	m := marshalutil.New(37)
+	m.WriteBytes(totalParticipationStakingKeyForEventPrefix(eventID)) // 33 bytes
```

Comment:
Did you forget the milestone index or same as above? Why is the milestone passed then?
```suggestion
	m.WriteBytes(totalParticipationStakingKeyForEventPrefix(eventID)) // 33 bytes
	m.WriteUint32(uint32(milestone))                                  // 4 bytes
```